src/server.c

```diff
@@ -3490,7 +3499,7 @@ static int shouldPropagate(int target) {
  * dbid value of -1 is saved to indicate that the called do not want
  * to replicate SELECT for this command (used for database neutral commands).
  */
-static void propagateNow(int dbid, robj **argv, int argc, int target) {
+static void propagateNow(int dbid, robj **argv, int argc, int target, int slot) {
     if (!shouldPropagate(target)) return;
```

Comment:
This shortcut will prevent that `clusterFeedSlotExportLinks` below is reached in some situations (no AOF configured, no replica)